src/app/shared/components/table/table.component.html

```diff
@@ -11,7 +11,11 @@
             <cdk-header-cell *cdkHeaderCellDef class="govuk-table__header" attr.data-selector="{{column.case_field_id}}-header">{{column.label}}</cdk-header-cell>
             <cdk-cell class="govuk-table__cell" *cdkCellDef="let row">
                 <span attr.data-selector="{{column.case_field_id}}-value" *ngIf="column.date_format">{{row.case_fields[column.case_field_id] | date: column.date_format:'+0000'}}</span>
-                <span attr.data-selector="{{column.case_field_id}}-value" *ngIf="!column.date_format">{{row.case_fields[column.case_field_id]}}</span>
+                <span attr.data-selector="{{column.case_field_id}}-value" *ngIf="!column.date_format">
```

Comment:
We should ideally be looking to place an ellipsis in the row using CSS rather than cutting the string down. we have no control over the viewport of the browser so 20 characters may still cause problems in smaller dimensions

Reply:
yer was more of "patch" solution rather than an actual fix
UR and UX should be involved before actual proceeding